Services/CO.CDP.OrganisationInformation/Constants.cs

```diff
@@ -0,0 +1,30 @@
+namespace CO.CDP.OrganisationInformation;
+public static class Constants
```

Comment:
Let's name it with something that reflects what it's for, i.e. `IdentifierSchemes`.

Reply:
done